src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -1352,30 +1360,114 @@ def create_lvs(volume_group, parts=None, size=None, name_prefix='ceph-lv'):
 #
 ###########################################################
 
-PV_FIELDS = 'pv_name,pv_tags,pv_uuid,vg_name,lv_uuid'
-VG_FIELDS = 'vg_name,pv_count,lv_count,snap_count,vg_attr,vg_size,vg_free,vg_free_count'
-LV_FIELDS = 'lv_tags,lv_path,lv_name,vg_name,lv_uuid,lv_size'
+def convert_filters_to_str(filters):
```

Comment:
Why move those constants?

Reply:
Only because it's convention to declare constants at the top of file.